honeybee_radiance_command/options/gensky.py

```diff
@@ -0,0 +1,251 @@
+# coding: utf-8
+
+from .optionbase import (
+    OptionCollection,
+    BoolOption,
+    NumericOption,
+    TupleOption,
+)
+import warnings
+
+
+class GenskyOptions(OptionCollection):
+    """gensky command options.
+
+    Also see: https://floyd.lbl.gov/radiance/man_html/gensky.1.html
+    """
+
+    __slots__ = (
+        "_s",
+        "_c",
+        "_i",
+        "_u",
+        "_g",
+        "_b",
+        "_B",
+        "_r",
+        "_R",
+        "_t",
+        "_a",
+        "_o",
+        "_m",
+        "_ang")
+
+    def __init__(self):
+        """Gensky command options."""
+
+        OptionCollection.__init__(self)
+
+        self._s = BoolOption("s", "Sunny sky without sun - default: False")
+        self._c = BoolOption("c", "Cloudy sky - default: False")
+        self._i = BoolOption("i", "Intermediate sky without sun - default: False")
+        self._u = BoolOption("u", "Uniform cloudy sky - default : False")
+        self._g = NumericOption("g", "Average ground reflectance")
+        self._b = NumericOption("b", "Zenith brightness")
+        self._B = NumericOption("B", "Zenith brightness")
```

Comment:
It was really strange that both of them have the exact same name.

```suggestion
        self._B = NumericOption("B", "Same as -b, except zenith brightness is computed from the horizontal diffuse irradiance")
```